Makefile

```diff
@@ -18,6 +18,9 @@ endif
 
 PYTHON_DIR = py
 PYTHON_TESTS = $(PYTHON_DIR)/tests
+PIP = $(PYTHON_BIN)/pip
```

Comment:
Do we need pip here? I can not see it being used

Reply:
It is used when you pull in the external test library.